src/ext/java/org/opentripplanner/ext/transmodelapi/TransmodelGraphQLSchema.java

```diff
@@ -623,6 +623,12 @@ private GraphQLSchema create() {
                         .type(Scalars.GraphQLFloat)
                         .defaultValue(routing.request.walkReluctance)
                         .build())
+                .argument(GraphQLArgument.newArgument()
+                        .name("waitReluctance")
+                        .description("Wait cost is multiplied by this value. Setting this to a value lower than 1 indicates that waiting is better than staying on a vehicle.")
+                        .type(Scalars.GraphQLFloat)
```

Comment:
Include that this should never be set higher than walkReluctance since that would lead to walking down the line to avoid waiting.